src/interface.rs

```diff
@@ -75,6 +75,27 @@ ast_types! {
                 semi_colon: term!(;),
             }),
         }),
+        /// Parses an async iterable declaration `[attributes]? async (iterable<attributedtype> | iterable<attributedtype, attributedtype>) (( args ))? ;`
+        AsyncIterable(enum AsyncIterableInterfaceMember<'a> {
+            /// Parses an async iterable declaration `[attributes]? async iterable<attributedtype> (( args ))? ;`
+            Single(struct SingleTypedAsyncIterable<'a> {
+                attributes: Option<ExtendedAttributeList<'a>>,
+                r#async: term!(async),
```

Comment:
`async` is a keyword in Rust, so I had to resort to a raw identifier. Alternative solutions are welcome. 🙂

Reply:
Found [a better solution from the `long long` type](https://github.com/rustwasm/weedle/blob/c7374a5ae0f3af0a659cfbdd7305b78b1404f4e8/src/types.rs#L87). 😉